diff -urNp old/src/libjasper/include/jasper/jas_math.h new/src/libjasper/include/jasper/jas_math.h
--- old/src/libjasper/include/jasper/jas_math.h	2017-05-25 12:33:43.358369488 +0200
+++ new/src/libjasper/include/jasper/jas_math.h	2017-05-25 12:35:02.254994460 +0200
@@ -86,6 +86,9 @@
 
 #ifdef __cplusplus
+#ifndef SIZE_MAX
+#define SIZE_MAX ((size_t) - 1)
+#endif
 extern "C" {
 #endif
 
 /******************************************************************************\
